android/sdl_android/src/main/java/com/smartdevicelink/transport/utl/SdlDeviceListener.java

```diff
@@ -177,6 +180,48 @@ public void handleMessage(@NonNull Message msg) {
                     break;
             }
         }
+// TODO determine why do we need this
```

Comment:
This section will read vehicle type info if available and close BT socket. Please check [here](https://raw.githubusercontent.com/smartdevicelink/sdl_evolution/master/assets/proposals/0293-vehicle-type-filter/OEM_APPS_2.png).